src/lib/components/icon/Icon.component.js

```diff
@@ -0,0 +1,136 @@
+import React, { Suspense } from "react";
+import styled, { css } from 'styled-components';
+import { getTheme } from '../../utils';
+import Loader from '../loader/Loader.component';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+
+export const iconTable = {
+  "Account": "fas faWallet",
+  "Backend": "fas faNetworkWired",
+  "Tape": "fas faTape",
+  "Node-backend": "fas faServer",
+  "Volume-backend": "fas faHdd",
+  "Node-pdf": "fas faDatabase",
+  "Volume-pdf": "fas faCompactDisc",
+  "Network": "fas faProjectDiagram",
+  "Bucket": "fas faGlassWhiskey",
+  "Cloud-backend": "far faCloud",
```

Comment:
This icon seems not loading

Reply:
The Cloud icon required a PRO font awesome

Reply:
Yes, fully correct. 
Let's use the free version now, we might have our own SVG in a mid-term objective.

Reply:
Done